### DISCUSSION TITLE
Mengurutkan highchart (column) berdasarkan group

### DISCUSSION THREAD

**Discussion by jeremiaww** [teamalgoritma/community#93] [Q&A] [answered]
Hello!

Izin bertanya bagaimana cara dalam mengurutkan highchart (column) berdasarkan group = age.

Tujuannya agar tampilannya menjadi lebih bagus.

<img width="951" alt="image" src="https://user-images.githubusercontent.com/99194718/167762288-37a59a9c-d52d-4c61-89b2-1c8641c346aa.png">

<img width="306" alt="image" src="https://user-images.githubusercontent.com/99194718/167762677-8ddea4b3-b45c-461b-893e-8b2353714a99.png">

Urutan age 5-14 years, harapannya mau di paling pertama.
Ketika di arrange berdasarkan age, urutan dari 5-14 years berada di tengah.

Terima kasih.

**Comment by VictorNugraha** [comment, marked as answer]
Hallo, @jeremiaww!

Mungkin bisa dilihat lagi urutan levels dalam kolom _**age**_, apakah levelsnya sudah sesuai dengan urutan yang di-inginkan. Jika belom sesuai Bapak bisa memanfaatkan fungsi **_factor()_** untuk mengurutkan levels factor dalam sebuah kolom. Pada fungsi tersebut kita bisa isi dengan 2 parameter, yaitu:

- **_x_** -> nama kolom yang ingin diatur levels factornya.
- _**levels**_ -> urutan levels factor yang diharapkan.

Berikut contoh codenya:

```
df$nama_kolom<- factor(x = df$nama_kolom, levels = c("urutan_1", "urutan_2", "urutan_3"))
```

Demikian infromasi yang dapat saya sampaikan, semoga menjawab pertanyaan yang diberikan. Selamat mencoba :D

**Reply by jeremiaww** [reply]
Mantap, bisa mas.

Thanks a lot, again :)
